### PR TITLE
Add second lab.

### PR DESCRIPTION
**Pull request by Sassiq**
Add main elements of the page and README.md with the wireframe link.

### CODE REVIEW

**Comment by nastik1811** [comment]
Where is the wireframe link?

**Comment by Sassiq** [comment]
> Where is the wireframe link?

In the README.md - line 13

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+## Functions of our app:
+
+- Scroll throw new films
+- Buy tickets
+- View your bought tickets
+
+## Wireframe:
+
+[Desktop]("https://www.figma.com/community/file/1101608530288529132/Cinema-Website")
```
Comment (nastik1811) [inline README.md]: Remove quotes form the link.

**Comment by nastik1811** [comment]
Is `5` enough?

**Comment by Sassiq** [comment]
> Is `5` enough?

Added more functions in the README.md and removed quote from the link

**Comment by nastik1811** [comment]
You have not enough tags on wireframe

**Comment by Sassiq** [comment]
> Is `5` enough?

Sorry, there was a misunderstanding. I was thinking that 5 is a line of README.md, not a mark. If we are talking about mark it's acceptable)

**Review by nastik1811 — APPROVED** [review]

**Pull request merged**